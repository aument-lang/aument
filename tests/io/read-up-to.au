file = io::open("text.txt", "r");
print file.io::read_up_to(2),'\n';