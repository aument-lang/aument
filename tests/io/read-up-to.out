fopen text.txt, rb
getc
getc
aa
